MdePkg/Include/IndustryStandard/Acpi65.h

```diff
@@ -19,6 +19,15 @@
 //
 #pragma pack(1)
 
+///
+/// _STA bit definitions ACPI 6.5 s6.3.7

```

Comment:
Not here, but typo in the commit message: definations -> definitions

Reply:
Ack